acados/ocp_nlp/ocp_nlp_sqp_rti.c

```diff
@@ -693,10 +874,16 @@ int ocp_nlp_sqp_rti_precompute(void *config_, void *dims_, void *nlp_in_, void *
     for (ii = 0; ii < N; ii++)
     {
         // set T
-        config->dynamics[ii]->model_set(config->dynamics[ii], dims->dynamics[ii], nlp_in->dynamics[ii], "T", nlp_in->Ts+ii);
+        config->dynamics[ii]->model_set(config->dynamics[ii],
+            dims->dynamics[ii], nlp_in->dynamics[ii], "T", nlp_in->Ts+ii);
+
         // dynamics precompute
-        status = config->dynamics[ii]->precompute(config->dynamics[ii], dims->dynamics[ii], nlp_in->dynamics[ii],
-                                                  opts->nlp_opts->dynamics[ii], nlp_mem->dynamics[ii], nlp_work->dynamics[ii]);
+        status = config->dynamics[ii]->precompute(config->dynamics[ii],
+            dims->dynamics[ii], nlp_in->dynamics[ii],
+            opts->nlp_opts->dynamics[ii],
+            nlp_mem->dynamics[ii],
+            nlp_work->dynamics[ii]);
```

Comment:
ok trying to void too long lines, but this is a bit excessive

Reply:
well, that's the best I could do to stay within 80 lines... which is exactly how many lines fit on a vim vertical split on a 13 inch display with a reasonable resolution :p

Reply:
Was also thinking this.
I remember the lint rule was 100 characters per line, which I still try to stick to.

Reply:
it was 80 recommended, 100 max admitted, I think. But again, it's not a style exercise. They just don't fit on a 13 inch display in horizontal split. Try yourself!